ui/package.json

```diff
@@ -96,6 +96,8 @@
     "@babel/core": "^7.11.5",
     "@babel/plugin-proposal-class-properties": "^7.10.4",
     "@babel/plugin-proposal-decorators": "^7.10.5",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.12.1",
+    "@babel/plugin-proposal-optional-chaining": "^7.12.1",
```

Comment:
Optional chaining should be supported by default as of v7.8.0 in `@babel/preset-env`. We are running v7.11.5 according to the lock file. I wonder why it's not working...

https://babeljs.io/blog/2020/01/11/7.8.0

Reply:
I saw that, and yet the opposite seemed to happen.  Is it possible this decision was reversed at some point?

Reply:
Also, it wasn't clear if "supported" meant there's no need to require the plugin in the webpack config, or just that the package is automatically required.